api/types/plugin_static_credentials.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+import "github.com/gravitational/trace"
+
+// PluginStaticCredentials are static credentials for plugins.
+type PluginStaticCredentials interface {
+	// ResourceWithSecrets provides common resource methods.
```

Comment:
```suggestion
	// ResourceWithLabels provides common resource methods.
```
Is this supposed to be `ResourceWithLabels`?